test/py/onnx_backend_test.py

```diff
@@ -1,9 +1,12 @@
-import sys
+import sys, os
 if sys.version_info < (3, 0):
     sys.exit()
 
+# add the lib folder (containing migraphx) to sys.path
+sys.path.append(os.getcwd() + "/lib")
```

Comment:
CMake should set the correct python path when running the backend tests, so this shouldn't be added.